test/utils.js

```diff
@@ -114,7 +114,6 @@ const fixtures = {
 		const { rows:inviteRows }=  await db.query(
 			`INSERT INTO user_invites 
 			("email") VALUES ($1) RETURNING *`, [newData.email])
-			
```

Comment:
let's try not to have this file in the changes if there's no code change here

Reply:
yes please